clustering/server/src/main/java/org/wildfly/clustering/server/registry/CacheRegistry.java

```diff
@@ -148,9 +148,11 @@ public Group getGroup() {
 
     @Override
     public Map<K, V> getEntries() {
-        Set<Node> nodes = this.group.getNodes().stream().collect(Collectors.toSet());
         try (Batch batch = this.batcher.createBatch()) {
-            return this.cache.getAdvancedCache().getAll(nodes).values().stream().collect(Collectors.toMap(entry -> entry.getKey(), entry -> entry.getValue()));
+            // ISPN-8459 Replace with org.infinispan.AdvancedCache#getAll once reliable (misses entries on topology change)
+            return this.group.getNodes().stream()
```

Comment:
Just an update here: since registry is always a replicated cache, the getAll problem might not manifest here. We need to wait for results with QE (@mvinkler).

Reply:
Does this mean we should hold off merging until we get those results?

Reply:
The issue does not affect scenarios with a replicated cache. Sorry for late response.